modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContext.java

```diff
@@ -241,6 +245,27 @@ ddmFormInstance, _getHttpServletRequest(),
 						_ddmFormWebConfiguration.
 							maximumRepetitionsForUploadFields());
 				}
+
+				if (Objects.equals(
+						ddmFormInstance.getStorageType(), "object")) {
+
+					DDMFormInstanceSettings ddmFormInstanceSettings =
+						ddmFormInstance.getSettingsModel();
+
+					ObjectField objectField =
+						_objectFieldLocalService.getObjectField(
+							GetterUtil.getLong(
+								ddmFormInstanceSettings.objectDefinitionId()),
+							_getObjectFieldName(ddmFormField));
+
+					long objectFieldId = objectField.getObjectFieldId();
+
+					ddmFormField.setProperty("objectFieldId", objectFieldId);
+
+					ddmFormField.setProperty(
+						"objectFieldAcceptedFileExtensions",
+						_getObjectAcceptedFileExtensions(objectFieldId));
```

Comment:
@nicolas-sss we can remove the blank line and put the `objectFieldAcceptedFileExtensions` before `objectFieldId`:

```
ddmFormField.setProperty(
	"objectFieldAcceptedFileExtensions",
		_getObjectAcceptedFileExtensions(objectFieldId));
ddmFormField.setProperty("objectFieldId", objectFieldId);
```

Reply:
Done!